src/stream.c

```diff
@@ -259,7 +259,8 @@ int stream_enable_rx(struct stream *strm, bool enable)
 	rx_set_enable(strm->rx, true);
 
 	if (strm->rtp && strm->cfg.rxmode == RX_MODE_THREAD &&
-	    strm->type == MEDIA_AUDIO && !rx_running(strm->rx))
+	    strm->type == MEDIA_AUDIO && !stream_bundle(strm) &&
```

Comment:
If `RX_MODE_THREAD` is configured but disabled for other reasons there should be at least a info message why I think.

Reply:
okay. I'll add one.

In a final version we should make video thread safe also.